src/main/scala/me/rexim/morganey/autocompletion/ReplAutocompletion.scala

```diff
@@ -123,7 +112,13 @@ object ReplAutocompletion {
     filtered.getOrElse(allNames)
   }
 
-  private def lastNameInLine(line: String): Option[(Int, String)] = {
+
+  /** Takes the longest postfix of a string that matches the {{Language.identifier}} regex
```

Comment:
Instead of `postfix` I recommend using the word `suffix`. In my option this sounds better, but both variants are equally fine.